packages/rum-core/src/domain/resource/resourceCollection.spec.ts

```diff
@@ -316,6 +317,13 @@ describe('resourceCollection', () => {
     )
   })
 
+  it('should discard 0 status code', () => {
+    const { lifeCycle, rawRumEvents } = setupBuilder.build()
+    const performanceEntry = createPerformanceEntry(RumPerformanceEntryType.RESOURCE, { responseStatus: 0 })
+    lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [performanceEntry])
+    expect((rawRumEvents[0].rawRumEvent as RawRumResourceEvent).resource.status_code).toBe(undefined)
```

Comment:
/
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    expect((rawRumEvents[0].rawRumEvent as RawRumResourceEvent).resource.status_code).toBeUndefined()
```